model/user.go

```diff
@@ -22,6 +26,16 @@ func PasswordHash(password string) string {
 	return string(hashed)
 }
 
+// 保存前にメールアドレスを使用しAPIトークンとして生成してセットする
```

Comment:
comment on exported method User.BeforeSave should be of the form "BeforeSave ..."